include/aspect/parameters.h

```diff
@@ -307,6 +307,8 @@ namespace aspect
     double                         start_time;
     double                         CFL_number;
     double                         maximum_time_step;
+    double                         reaction_time_step;
+    int                            reaction_steps_per_advection_step;
```

Comment:
At the moment, I need it for the benchmark cases, because to demonstrate that the scheme works correctly, I vary the `reaction_time_step` and show convergence (in case you haven't looked at the pdf files yet -- github doesn't show them by default -- they are all convergence plots).